src/super_gradients/recipes/dataset_params/pascal_aug_segmentation.yaml

```diff
@@ -0,0 +1,52 @@
+train_dataset_params:
```

Comment:
Rename the file to pascal_aug_segmentation_dataset_params.yaml please (just as the rest...)